src/ensembl/src/content/app/browser/drawer/drawer-views/DrawerBookmarks.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FunctionComponent } from 'react';
+
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { RootState } from 'src/store';
+import { Bookmark } from 'src/content/app/browser/track-panel/trackPanelState';
+import { TrackStates } from 'src/content/app/browser/track-panel/trackPanelConfig';
+import { updateTrackStates } from 'src/content/app/browser/browserActions';
+import { closeTrackPanelModal } from 'src/content/app/browser/track-panel/trackPanelActions';
+import { getActiveGenomePreviouslyViewedObjects } from 'src/content/app/browser/track-panel/trackPanelSelectors';
+import { closeDrawer } from 'src/content/app/browser/drawer/drawerActions';
+import upperFirst from 'lodash/upperFirst';
+import styles from './DrawerBookmarks.scss';
```

Comment:
_(style issue)_ Can we keep some order in the order of imports? Imports from node modules in the top; imports from styles in the very bottom; imports from helpers, actions, selectors, components, etc. clustered in their own groups?